impl/client/src/main/javascript/web/pentaho/type/filter/_core/tree.js

```diff
@@ -193,6 +193,7 @@ define([
         props: [
           {
             name: "operands",
+            nameAlias: "args",
```

Comment:
I still find it confusing to have a name and a nameAlias that don't share a radical word.